diff --git a/src/PrusaSlicer.cpp b/src/PrusaSlicer.cpp
index 3d2e6e6b7..24e526d29 100644
--- a/src/PrusaSlicer.cpp
+++ b/src/PrusaSlicer.cpp
@@ -70,13 +70,6 @@ int CLI::run(int argc, char **argv)
     // Mark the main thread for the debugger and for runtime checks.
     set_current_thread_name("slic3r_main");
 
-#ifdef __WXGTK__
-    // On Linux, wxGTK has no support for Wayland, and the app crashes on
-    // startup if gtk3 is used. This env var has to be set explicitly to
-    // instruct the window manager to fall back to X server mode.
-    ::setenv("GDK_BACKEND", "x11", /* replace */ true);
-#endif
-
 	// Switch boost::filesystem to utf8.
     try {
         boost::nowide::nowide_filesystem();
